### PR TITLE
Add react searching custom sample

### PR DESCRIPTION
**Pull request by mddragnev**

### CODE REVIEW

File: samples/grids/grid/data-searching/package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "eject": "react-scripts eject",
+    "lint": "eslint ./src/**/*.{ts,tsx}"
+  },
+  "dependencies": {
+    "igniteui-react": "18.2.2-beta.0",
+    "igniteui-react-core": "18.2.2-beta.0",
+    "igniteui-react-datasources": "18.2.2-beta.0",
+    "igniteui-react-grids": "18.2.2-beta.0",
+    "igniteui-react-inputs": "18.2.2-beta.0",
+    "igniteui-react-layouts": "18.2.2-beta.0",
```
Comment (HUSSAR-mtrela) [inline samples/grids/grid/data-searching/package.json:22]: you need to target proget packages for all IG packagese, e.g.
"@infragistics/igniteui-react-grids": "23.2.152"

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: samples/grids/grid/data-searching/package.json (hunk 1 above)
Comment (mddragnev) [reply samples/grids/grid/data-searching/package.json:22]: Why is this? The proget packages require some manual work (copy the files outside of the scope in node modules). Those which are in the public npm works fine.

Comment (mddragnev) [reply samples/grids/grid/data-searching/package.json:22]: @HUSSAR-mtrela Also, this package does not contain the translated WC grid

Comment (HUSSAR-mtrela) [reply samples/grids/grid/data-searching/package.json:22]: @mddragnev ignore me previous request per discussion with Graham and Stamen in Teams

**Review by skrustev — APPROVED** [review]

**Review by dkamburov — APPROVED** [review]

**Pull request merged**